### ISSUE TITLE
Windows Hello On Ubuntu

### ISSUE THREAD

**Issue by Choccy-vr** [linux-surface/linux-surface#1635]
### Description
My Surface Go 2 has Windows Hello functionality but I want that in Ubuntu. I saw a program called Howdy! but I can't get the path it wants for the cameras (/dev/videoX) when I test it using something like ffplay it just says it is not a capture device. Is there any other way to get Windows Hello functionality or fix the problem I am having with Howdy!.

### Environment

- Hardware model: Surface Go 2
- Kernel version: 6.12.3
- Distribution: Ubuntu

### Errors
```
[video4linux2,v4l2 @ 0x76a5f4000cc0] Not a video capture device.
/dev/video0: No such device
```
Note: I tried video0-13 and all devices listed by this command `v4l2-ctl --list-devices`

**Comment by gonggy** [comment]
It seems that the camera driver is not installed.

**Comment by Choccy-vr** [comment]
> It seems that the camera driver is not installed.

I am pretty sure it is but just in case can you give me the link to the instructions.

**Comment by freechelmi** [comment]
Here https://github.com/linux-surface/linux-surface/wiki/Camera-Support

On Ubuntu 24.04 , that would be : 

sudo zstd -d -f /lib/firmware/intel/ipu3-fw.bin.zst

sudo apt install -y gstreamer1.0-libcamera pipewire-libcamera libcamera0.2 libspa-0.2-libcamera libcamera-v4l2  libcamera-tools  libcamera-ipa'

**Comment by Choccy-vr** [comment]
Nope that doesn't work. I ran the commands and it seemed to install fine. When checking the video paths I get all of these ``ipu3-imgu (PCI:0000:00:05.0):
	/dev/video4
	/dev/video5
	/dev/video6
	/dev/video7
	/dev/video8
	/dev/video9
	/dev/video10
	/dev/video11
	/dev/video12
	/dev/video13
	/dev/media1

Intel IPU3 CIO2 (PCI:0000:00:14.3):
	/dev/video0
	/dev/video1
	/dev/video2
	/dev/video3
	/dev/media0

``but when testing them in ffplay I get this error ``[video4linux2,v4l2 @ 0x755568000c80] Not a video capture device.
/dev/video13: No such device``

**Comment by freechelmi** [comment]
Yes I have the same problem , it only work on cheese because it use gstreamer , there is a missing piece here

**Comment by Choccy-vr** [comment]
So at this time there isn't really a way to get a video video without g streamer?

**Comment by freechelmi** [comment]
It works now on my machine I have video on native firefox , not yet on flatpak

Check all the packages I used here 

https://github.com/ekimiateam/linux-universal-driver/blob/f783ade28ad4b5eca29dcfd0760c1cb220153877/system76driver/actions.py#L1759

You also need that your user is in the video group
